app/kingdom/contest.py

```diff
@@ -0,0 +1,53 @@
+from datetime import date
+
+from app.people.knight import Knight
+
+
+class Contest:
+    def __init__(self, date: date, king: str, knights: dict) -> None:
+        self.date = date
+        self.king = king
+        self.knights = knights
+        print(f"On the {date} Fight contest of {len(knights)} "
+              f"knights was created by King {king}")
```

Comment:
The print statement uses a mix of single and double quotes. According to the checklist, you should use double quotes consistently throughout the code.